src/nupic/regions/PyRegion.py

```diff
@@ -177,6 +177,15 @@ def getOutputElementCount(self, name):
     name: the name of the output
     """
 
+  def getAlgorithm(self):
```

Comment:
I think this is one of those Pythonish-hacky things that is perhaps not clean but would be extremely useful in practice. This request seems to come up all the time.